layouts/partials/contributor-card.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="contributor-card">
+<div class="contributor-card" style="display: none;">
```

Comment:
Should this be `visibility: hidden` instead of `display: none`?

Reply:
But there's a problem: The "div" space is still left, so are you sure you want this? Something like:
![1](https://user-images.githubusercontent.com/52018749/113227697-deda6780-92c5-11eb-8cdf-0694fef829cf.png)

IMO: If it fails to load, the space shouldn't be shown....

Reply:
Ah, yeah, I can see the merit of `display:none` in that case.